Compatibility update on changelog and upgrade notice

gertrude-paystack opened the pull request:
Compatibility changes made to the changelog and upgrade notice

lukman-paystack reviewed (commented):
The new version number should be 1.0.4 not 1.1.0

The first number is for major changes, the second number is for minor changes and the third number is for patches. In this case, we are only changing the version number and making no edits to the plugin so it should be 1.0.4

You can read more on semantic versioning here https://semver.org/

lukman-paystack reviewed (commented):
Include your name in the contributors list

lukman-paystack reviewed (changes requested):
Add your name to the contributors list and change the version number

gertrude-paystack committed:
Initial commit: Paystack Events Calendar v1.0.7 base

gertrude-paystack committed:
PATCH v1.1.0: Update version and comprehensive changelog

- Update plugin version from 1.0.7 to 1.1.0
- Add detailed changelog covering all major fixes:
  * Currency registration (NGN, GHS, KES, ZAR, XOF, EGP)
  * API key saving functionality
  * Debug logging implementation
  * Form structure improvements
  * WordPress hooks compatibility
